http_check/datadog_checks/http_check/config_models/defaults.py

```diff
@@ -40,10 +40,6 @@ def instance_empty_default_hostname():
     return False
 
 
-def instance_http_response_status_code():
-    return '(1|2|3)\\d\\d'
```

Comment:
I was confused but we don't use it so in case someone else is wondering why we can drop this: https://github.com/DataDog/integrations-core/blob/24feb58fcc829a7b9d0a4a1ce801796d7c372b02/http_check/datadog_checks/http_check/config.py#L47

Reply:
Good catch, that was completely unintentional. I may still try to get that back in just in case.

Reply:
I managed to put it back.